crates/wasi-nn/examples/classification-component-onnx/src/lib.rs

```diff
@@ -0,0 +1,131 @@
+cargo_component_bindings::generate!();
```

Comment:
Good to see the auto-generated bindings work!

Reply:
I updated this to reflect the latest changes in `cargo component` and preview 2.